src/app/pages/LendingPage/components/CurrencyContainer/CurrencyRow.tsx

```diff
@@ -55,6 +55,7 @@ const CurrencyRow: React.FC<Props> = ({
             className="tw-block tw-w-full tw-mb-3 tw-rounded-lg tw-bg-primary-25 hover:tw-opacity-75"
             textClassName="tw-text-base"
             disabled={depositLocked || !connected}
+            data-action-id={`lend-deposit-button-${asset}`}
```

Comment:
```suggestion
            data-action-id={`lend-depositButton-${asset}`}
```
This was defined incorrectly on the spec